src/styles/block-style/page-style.scss

```
@@ -0,0 +1,7 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
```

```suggestion
  margin: 0;
  font-family: "Roboto", sans-serif;
```
Consider adding a font family here to get it inherited by other elements